spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -27,15 +27,31 @@
   end
 
   describe "::call" do
+    let(:user) { create(:user, :with_identity, identities: ["github"], created_at: 1.week.ago) }
+
     it "does not send a notification to an unsubscribed user" do
-      user = create(:user, :with_identity, identities: ["github"], created_at: 1.week.ago, welcome_notifications: false)
+      user.update!(welcome_notifications: false)
       expect do
         sidekiq_perform_enqueued_jobs { described_class.call(user.id) }
       end.to not_change(user.notifications, :count)
     end
 
-    it "sends only 1 notification at a time, in the correct order" do # rubocop:disable RSpec/MultipleExpectations, RSpec/ExampleLength
-      user = create(:user, :with_identity, identities: ["github"], created_at: 1.day.ago)
+    it "does not send a notification and if no active broadcast exists" do
+      welcome_broadcast.update!(active: false)
```

Comment:
can't change the record here because of `let_it_be_readonly`. We could change that to `let_it_be` then it should work

Reply:
literally pushed up the commit to fix this as soon while you wrote this comment 😛